public/app/screens/main/main-screen.jsx

```diff
@@ -7,12 +7,21 @@ var Navigation = require('common/regions/navigation');
 var style = require("./style.scss");
 
 var MainScreen = React.createClass({
+    
+    getInitialState: function() {
+        return {
+            navItems: [
```

Comment:
Create a store.
If you wish you can merge it, but create an issue with the code reference.